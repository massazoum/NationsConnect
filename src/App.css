
body {
  background-image: url(./images/bgmain.png);
  font-family: 'Lato', sans-serif;
  color: #ffff;
}
