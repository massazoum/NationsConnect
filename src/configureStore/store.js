import { configureStore } from '@reduxjs/toolkit';
import RegionReducer from '../reducers/CountriesReducer';

const store = configureStore({
  reducer: {
    region: RegionReducer,
  },
});

export default store;
