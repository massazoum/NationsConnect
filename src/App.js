import './App.css';
import Home from './components/HomePage';
import Details from './components/DetailsPage';

function App() {
  return (
    <div className="App">
      <Home />
      <Details />
    </div>
  );
}

export default App;
