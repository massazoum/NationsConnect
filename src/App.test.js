import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from './configureStore/store';
import App from './App';

describe('App', () => {
  it('Testing of App.js', () => {
    const app = render(
      <Provider store={store}>
        <App />
      </Provider>,
    );
    expect(app).toMatchSnapshot();
  });
});
