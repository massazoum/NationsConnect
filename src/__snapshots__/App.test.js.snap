// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Testing of App.js 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div>
          <div
            class="mainHome"
          >
            <div
              class="searchBar"
            >
              <div>
                <img
                  alt=""
                  height="20px"
                  src="arrow.png"
                  width="20px"
                />
              </div>
              <div>
                Welcom to you
              </div>
              <div
                class="iconSearch"
              >
                <div>
                  <img
                    alt=""
                    height="20px"
                    src="mic.png"
                    width="20px"
                  />
                </div>
                <div>
                  <img
                    alt=""
                    height="20px"
                    src="setting.png"
                    width="20px"
                  />
                </div>
              </div>
            </div>
            <div
              class="mainItem"
            >
              <div
                class="AfricaImg"
              >
                <img
                  alt=""
                  height="150px"
                  src="africa.png"
                  width="100px"
                />
              </div>
              <div
                class="AfricaText"
              >
                <h1>
                  Africa
                </h1>
              </div>
            </div>
            <div
              class="africa_countries"
            >
              <div
                class="africa_countriesText"
              >
                List of africa countries  
              </div>
              <div>
                Search
                <input
                  placeholder="Name country"
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              class="items"
            />
          </div>
        </div>
        <div />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div>
        <div
          class="mainHome"
        >
          <div
            class="searchBar"
          >
            <div>
              <img
                alt=""
                height="20px"
                src="arrow.png"
                width="20px"
              />
            </div>
            <div>
              Welcom to you
            </div>
            <div
              class="iconSearch"
            >
              <div>
                <img
                  alt=""
                  height="20px"
                  src="mic.png"
                  width="20px"
                />
              </div>
              <div>
                <img
                  alt=""
                  height="20px"
                  src="setting.png"
                  width="20px"
                />
              </div>
            </div>
          </div>
          <div
            class="mainItem"
          >
            <div
              class="AfricaImg"
            >
              <img
                alt=""
                height="150px"
                src="africa.png"
                width="100px"
              />
            </div>
            <div
              class="AfricaText"
            >
              <h1>
                Africa
              </h1>
            </div>
          </div>
          <div
            class="africa_countries"
          >
            <div
              class="africa_countriesText"
            >
              List of africa countries  
            </div>
            <div>
              Search
              <input
                placeholder="Name country"
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="items"
          />
        </div>
      </div>
      <div />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
